File: app/src/main/java/io/mgba/Controllers/UI/Fragments/Main/FavouritesFragment.java
package io.mgba.Controllers.UI.Fragments.Main;

import io.mgba.Controllers.UI.Fragments.Main.Interfaces.BaseGameFragment;
import io.mgba.R;

public class FavouritesFragment extends BaseGameFragment {

    @Override
    protected void prepareDrawables() {
        noContentImage.setImageDrawable(getResources().getDrawable(R.drawable.ic_favorite_grey_500_48dp));
        noContentMessage.setText(R.string.no_favourites);
    }
}

File: app/src/main/java/io/mgba/main/fragments/FavouritesFragment.kt
package io.mgba.main.fragments

import androidx.core.content.ContextCompat
import io.mgba.R
import kotlinx.android.synthetic.main.games_list_view.*

class FavouritesFragment : GameFragment() {

    override fun prepareDrawables() {
        noContentIcon.setImageDrawable(ContextCompat.getDrawable(context!!, R.drawable.ic_favorite))
        noContentMessage.setText(R.string.no_favourites)
    }
}
